components/ProjectCard/index.js

```diff
@@ -38,26 +52,31 @@ const CardContainer = styled.div`
   border-radius: 20px;
   position: relative;
   overflow: hidden;
-  width: 350px;
-  height: 200px;
   display: flex;
   flex-direction: row;
   list-style: none;
   padding: 0;
-  margin: 0;
   gap: 1rem;
   transition: all 0.5s ease;
+  z-index: 0;
+  height: 100%;
   &:hover {
     transform: scale(1.1);
   }
 `;
 
+const CardWrapper = styled.div`
+  width: 350px;
+  height: 200px;
+  position: relative;
+  padding: 0;
+`;
+
 const StyledImage = styled(Image)`
   width: 50%;
   height: auto;
   border-radius: 10px 0 0 13px;
   object-fit: cover;
-  z-index: 2;
   margin: 1rem;
```

Comment:
Why do you use rem for margin? rem is a good possibility for making websites more accessibility, but the margin and padding don't have to be bigger, if the default font-size of the browser is bigger, do they?

Reply:
You're right but since we don't plan to change the root font size, we just use rem or % for everything. Force of habit :)